nilearn/_utils/tags.py

```diff
@@ -0,0 +1,57 @@
+"""Nilearn tags for estimators."""
```

Comment:
Maybe give a short explanation or reference to explain what tags are and why they're needed